src/elements/element.arc.js

```diff
@@ -92,7 +92,7 @@ module.exports = Element.extend({
 		ctx.save();
 
 		ctx.beginPath();
-		ctx.arc(vm.x, vm.y, vm.outerRadius - pixelMargin, sA, eA);
+		ctx.arc(vm.x, vm.y, Math.max(vm.outerRadius - pixelMargin, 0), sA, eA);
```

Comment:
@nagix I needed this otherwise `borderAlign: 'inner'` on a `0` value in a polarArea chart crashes with a negative radius. Is there a better way to handle this?

Reply:
Thanks for catching this! I think this fix is fine.